client/src/javascript/components/sidebar/DiskUsage.js

```diff
@@ -0,0 +1,74 @@
+import {FormattedMessage} from 'react-intl';
+import React from 'react';
+
+import EventTypes from '../../constants/EventTypes';
+import DiskUsageStore from '../../stores/DiskUsageStore';
+import Size from '../general/Size';
+import Tooltip from '../general/Tooltip';
+import ProgressBar from '../general/ProgressBar';
+
+const METHODS_TO_BIND = ['onDiskUsageChange', 'getDisks'];
+
+export default class DiskUsage extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      disks: DiskUsageStore.getDiskUsage(),
+    };
+
+    METHODS_TO_BIND.forEach(method => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  componentDidMount() {
+    DiskUsageStore.listen(EventTypes.DISK_USAGE_CHANGE, this.onDiskUsageChange);
+  }
+
+  componentWillUnmount() {
+    DiskUsageStore.unlisten(EventTypes.DISK_USAGE_CHANGE, this.onDiskUsageChange);
+  }
+
+  onDiskUsageChange() {
+    this.setState({disks: DiskUsageStore.getDiskUsage()});
+  }
```

Comment:
I just merged a change which introduces a new pattern for getting data from stores. Please implement that here as well.

[Here's an example](https://github.com/Flood-UI/flood/blob/master/client/src/javascript/app.js#L94-L106) of how the higher order component can be used. After you implement it, you can just consume the data from the component props.